assets/manifests/slzb-06m.json

```diff
@@ -0,0 +1,36 @@
+{
+	"product_name": "SMLIGHT SLZB-06M",
+	"baudrates": {
+		"bootloader": [115200],
+		"cpc": [460800, 115200, 230400],
+		"ezsp": [115200],
+		"spinel": [460800]
+	},
+	"usb_filters": [
+		{
+			"pid": 60000,
+			"vid": 4292
+		}
+	],
+	"firmwares": [
+		{
+			"name": "Zigbee (EZSP)",
+			"url": "https://raw.githubusercontent.com/darkxst/silabs-firmware-builder/main/firmware_builds/slzb-06m/ncp-uart-hw-v7.4.1.0-slzb-06m-115200.gbl",
+			"type": "ncp-uart-hw",
+			"version": "7.4.1.0"
+		},
+		{
+			"name": "Multi-PAN (RCP)",
+			"url": "https://raw.githubusercontent.com/darkxst/silabs-firmware-builder/main/firmware_builds/slzb-06m/rcp-uart-802154-v4.4.1-slzb-06m-460800.gbl",

```

Comment:
Multipan firmware must use v7.3.1 for the time being. All firmware since then are broken.

Reply:
corrected here https://github.com/darkxst/silabs-firmware-builder/pull/64

Reply:
corrected